src/functionalTest/resources/application-functional.yaml

```diff
@@ -1,6 +1,6 @@
 targets:
   instance: ${TEST_URL:http://localhost:8088}
-  camunda: ${CAMUNDA_URL:http://camunda-bpm/engine-rest}
+  camunda: ${CAMUNDA_URL:http://camunda-local-bpm/engine-rest}
```

Comment:
CAMUNDA_URL on local env is set to camunda-local-bpm. I think it should be reverted.